_includes/seo.html

```diff
@@ -4,13 +4,20 @@
 {%- endif -%}
 {%- assign seo_url = seo_url | default: site.github.url -%}
 
-{% assign title_separator = site.title_separator | default: '-' | replace: '|', '&#124;' %}
```

Comment:
This may not be what's intended. The point here is to give a default value in the absence of `site.title_separater`. Putting this line inside an `{% if %}` statement defeats the current purpose.